pkg/autoscaler/autoscaler.go

```diff
@@ -212,12 +215,24 @@ func (a *Autoscaler) Scale(ctx context.Context, now time.Time) (int32, bool) {
 		return 0, false
 	}
 
-	// Log system totals
+	// Log system totals based on last reported stats per pod
 	totalCurrentQPS := int32(0)
 	totalCurrentConcurrency := float64(0)
 	for _, stat := range lastStat {
 		totalCurrentQPS = totalCurrentQPS + stat.RequestCount
 		totalCurrentConcurrency = totalCurrentConcurrency + stat.AverageConcurrentRequests
+
+		if a.EnablePodScopeMetrics {
+			// Log pod metrics
+			if !stat.Time.Add(30 * time.Second).After(now) {
+				// Zero out pod metrics after 30 seconds of no activity
+				a.reporter.Report(PodQpsM, 0.0, tag.Insert(podTagKey, stat.PodName))
+				a.reporter.Report(PodConcurrencyM, 0.0, tag.Insert(podTagKey, stat.PodName))
+			} else {
+				a.reporter.Report(PodQpsM, float64(stat.RequestCount), tag.Insert(podTagKey, stat.PodName))
+				a.reporter.Report(PodConcurrencyM, float64(stat.AverageConcurrentRequests), tag.Insert(podTagKey, stat.PodName))
```

Comment:
Why are we not emitting those from the queue-proxies themselves? Security/multi-tenancy considerations?